### PR TITLE
PLANET-4351: CPP - External link symbols missing

### PR DESCRIPTION
**Pull request by pablocubico**
Ref: https://jira.greenpeace.org/browse/PLANET-4351

### CODE REVIEW

File: assets/src/js/external_links.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
-jQuery(function($) {
+export const setupExternalLinks = function($) {
   let siteURL = window.location.host;
 
+  console.log(siteURL);
```
Comment (comzeradd) [inline assets/src/js/external_links.js]: Can you remove that? :D

Comment (pablocubico) [reply assets/src/js/external_links.js]: @comzeradd removed!

**Comment by comzeradd** [comment]
Can you also adjust CI eslint configuration, since js code is now in a different path?
https://github.com/greenpeace/planet4-master-theme/blob/planet-4351/.circleci/config.yml#L66

**Comment by pablocubico** [comment]
@comzeradd Paths fixed and linting issues solved!

**Pull request merged**